public/app/features/explore/LogsMetaRow.tsx

```diff
@@ -71,7 +71,7 @@ export const LogsMetaRow: React.FC<Props> = React.memo(
         label: 'Your logs might have incorrectly escaped content',
         value: (
           <Tooltip
-            content="We suggest to try to fix the escaping of your log lines first. This is an experimental feature, your logs might not be correctly escaped."
+            content="Attempt to automatically fix incorrectly escaped newline and tab sequences in log lines. Manually review the results to confirm that the replacements are correct."
```

Comment:
```suggestion
            content="Fix incorrectly escaped newline and tab sequences in log lines. Manually review the results to confirm that the replacements are correct."
```

Reply:
Thanks! I waffled on whether to be this explicit about the feature's reliability.

Reply:
"Attempt to automatically fix" does not read well, it can also reduce user confidence in the feature capability.

The preceding label says: "'Your logs might have incorrectly escaped content'. So, perhaps say: "Automatically fix incorrectly ..."

Reply:
I like @achatterjee-grafana's edit here.